src/App.js

```diff
@@ -3,11 +3,33 @@ import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 
 function App() {
+  const todoList = [
+    {
+      id: 1234,
+      title: "Complete CTD assignment",
+    },
+    {
+      id: 4321,
+      title: "Go grocery shopping",
+    },
+    {
+      id: 9876,
+      title: "Go the gym",
+    },
+    {
+      id: 4987,
+      title: "Do laundry",
+    },
+  ];
```

Comment:
Oops I circled back and noticed this. It's actually better off where you had it before in `TodoList.js`, reason being you basically want each component to have one job. `App.js` is the app, so it's job is to display all of the app's pieces. `TodoList.js` job is to display the todoList, so rather than storing the array in `App.js` and passing it through props to `TodoList.js`, it's better to just store it where it's being used.

Reply:
Thank you Geo. I will make the updates. 

I would really appreciate setting up some time to meet to talk more about props. I think hearing it explain a different way would help. I am on Pacific Time and work full time Monday through Friday 8:00 - 5:00 PM. I could meet tomorrow after work at 6:00 PM Pacific or maybe Sunday or Monday/Tuesday next week?

I appreciate your help.